staking-pool/README.md

```diff
@@ -298,3 +298,140 @@ pub fn update_reward_fee_fraction(&mut self, reward_fee_fraction: RewardFeeFract
+
+#### Upgrade to the latest near-shell:
+```bash
+npm install -g near-shell
+```
+#### Set Environment and Login:
+
+##### If not logged into the browser, recover your account with the seed phrase first
+https://wallet.betanet.nearprotocol.com/create/
+
+```bash
+#Set the NEAR environment to the target network (betanet,testnet,mainnet)
+export NEAR_ENV=betanet
+
+near login
+```
+
+#### Unstake and Withdraw:
+```bash
+#If you staked to your validator unstake, there is no withdraw
+near stake nearkat.betanet <staking public key> 0
+
+#If you staked to a contract get the staked balance
+near view my_validator get_account_staked_balance '{"account_id": "user1"}'
+
+#Unsake by copying and pasting the staked balance
+near call my_validator unstake '{"amount": "100000000000000000000000000"}' --accountId user1
+
+#Wait 4 epochs (12 hours) to withdraw and check if balance is available to withdraw
+near view my_validator is_account_unstaked_balance_available '{"account_id": "user1"}'
+
+#If is_account_unstaked_balance_available returns "true" withdraw
+near call my_validator withdraw '{"amount": "100000000000000000000000000"}' --accountId user1
+```
+#### Download new contract with Git:
+```bash
+mkdir staking-pool
+
+cd staking-pool
+
+git clone https://github.com/near/initial-contracts
+
+cd initial-contracts
+
+cd staking-pool
+```
+#### Build contract with Rust (This step is optional since the contract is compiled):
+##### Install Rust:
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+#Add rust to current shell path
+source $HOME/.cargo/env
+```
+##### Add wasm target to your toolchain:
+```bash
+rustup target add wasm32-unknown-unknown
+```
+##### Build:
+```bash
+./build.sh
+```
+#### Get validator key:
+```bash
+cat ~/.near/betanet/validator_key.json |grep "public_key"
+```
+#### Create a new account to deploy contract to
+- Set my_validator to the name you want publicly displayed
+- --masterAccount is your account you signed up to StakeWars2 with
+```bash
+near create_account my_validator --masterAccount=owner
+```
+#### Deploy the contract to the new account
+```bash
+near deploy --accountId=my_validator --wasmFile=res/staking_pool.wasm
+```
+#### Initialize staking pool at account `my_validator` for the owner account ID `owner`, given staking pool and 10% reward fee
+```bash
+near call my_validator new '{"owner_id": "owner", "stake_public_key": "CE3QAXyVLeScmY9YeEyR3Tw9yXfjBPzFLzroTranYtVb", "reward_fee_fraction": {"numerator": 10, "denominator": 100}}' --account_id owner
+```
+
+#### Check the current `seat price` to transfer the correct amount to your delegator(s)
+```bash
+near validators next| grep "seat price"
+```
+#### Register a delegator account (repeat these steps for additional delegators)
+-- https://wallet.betanet.near.org
+-- backup your seed phrase
+-- transfer NEAR from your MasterAccount to the delegator account
+
+#### Login and authorize the delegator
+```bash
+export NEAR_ENV=betanet
```

Comment:
Export is not needed for the 2nd time. It's already set to `betanet`

Reply:
Should I still work on the PR? Stefano had said he moved most of into StakeWars README. I still think it is viable and users are asking for more of a detailed approach.

Reply:
Yep. Let's merge it here